### PR TITLE
Popup messages using toastr

### PR DESCRIPTION
**Pull request by adenysen994**
Show all errors and pop-up messages using Toastr

### CODE REVIEW

File: front-end/src/components/Admin/Record/Client/index.js [hunk 1]
```diff
@@ -91,14 +92,16 @@ class SimpleRecord extends Component {
...
+        toastr.success('Данные успешно сохранены', response.status);
       })
       .catch(error => {
-        console.log(error);
-        this.showMessage('error', 'Ошибка при сохранении');
-        this.hideMessageAfterTimeout();
+        this.setState({
+          loading: false
+        });
+        toastr.error('шибка при сохранении', error);
```
Comment (Thealoner) [inline front-end/src/components/Admin/Record/Client/index.js]: "Ошибка"

File: front-end/src/components/Manager/Main/EntitiesMenu/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (Thealoner) [inline front-end/src/components/Manager/Main/EntitiesMenu/index.js]: Something is not right. I see a few files copied from Admin component to Manager component. They do not belong here. Please remove them if you added them by mistake.

**Comment by adenysen994** [comment]
[Show all errors and pop-up messages using Toastr](https://trello.com/c/DssHLCzY/242-show-all-errors-and-pop-up-messages-using-toastr)

File: front-end/src/components/Manager/Main/EntitiesMenu/index.js (hunk 2 above)
Comment (adenysen994) [reply front-end/src/components/Manager/Main/EntitiesMenu/index.js]: There are only necessary fixes. I didn't copy any files from Admin to Manager.
In manager component only replace console.log with toastr.
If i missed something please show which files.

Comment (Thealoner) [reply front-end/src/components/Manager/Main/EntitiesMenu/index.js]: Ah, I see. You moved them into 'Main' folder. For some reason they appear here as new files.

**Pull request merged**